pom.xml

```diff
@@ -143,7 +143,8 @@ LICENSE file.
     <openjpa.jdbc.version>2.1.1</openjpa.jdbc.version>
     <orientdb.version>2.2.37</orientdb.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <redis.version>2.9.0</redis.version>
+    <redis.version>3.2.0</redis.version>
+    <redisJson.version>1.1.1-SNAPSHOT</redisJson.version>
```

Comment:
we need a published dependency version. is there something specific to the 1.1.1 release this change set needs?

Reply:
This requires the reflective branch as outliend in the README.md of the subdir otherwise the redisjon i/f code will not work. Instructions on how to get and build this resulting in a jar in local Maven repo are also in the README.md

Reply:
When will a release happen with the needed changes?

Reply:
Do not know as this is beyond my control. I may be able to recheck.